app/models/game.rb

```diff
@@ -1,19 +1,3 @@
 class Game < ApplicationRecord
   belongs_to :user
```

Comment:
Hi Soo, does this line not need the  .... has_many :bookings, dependent: :destroy

Reply:
Never mind, it has it on the other commits..